edi_sale_ubl_oca/demo/edi_exchange_type.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+<!-- TODO: noupdate OR better create via post init hook   -->
+    <record id="demo_edi_exc_type_order_response_out" model="edi.exchange.type">
+        <field name="backend_type_id" ref="edi_ubl_oca.edi_backend_type_ubl" />
+        <field name="name">Demo UBL Sale Order Response</field>
+        <field name="code">demo_UBL_SaleOrderResponse_out</field>
+        <field name="direction">output</field>
+        <field name="exchange_filename_pattern">{record_name}-{type.code}-{dt}</field>
+        <field name="exchange_file_ext">xml</field>
```

Comment:
Did we miss `backend_id`?

Reply:
I don't know why simone didn't declare backend_id here but added it in [unit test](https://github.com/thienvh332/edi/blob/2b961b6a40f11f33214821200ff5d03bc9ddebf7/edi_sale_ubl_oca/tests/common.py#L65-L66) in the same commit.

Reply:
Please ignore
Actually, if backend is not defined, `create_record` will be based on `backend_type_id` ([here](https://github.com/OCA/edi/blob/e9fe1a70f7db810dfd5da879b8f25a15f490951a/edi_oca/models/edi_backend.py#L187)), so ok for me

Reply:
But take note that: `Auto` won't be used if not defined `backend_id` ([here](https://github.com/camptocamp/edi-framework/blob/4e14fb3d03abc7e500c31dd4165209437fb46783/edi_exchange_type_auto/models/edi_auto_exchange_consumer_mixin.py#L113))